src/components/admin/form-components/SearchBar.jsx

```diff
@@ -97,23 +127,25 @@ export class SearchBar extends React.Component {
 
 SearchBar.propTypes = {
   handleClick: PropTypes.func.isRequired,
-  suggestions: PropTypes.arrayOf(
-    PropTypes.shape({
-      title: PropTypes.string.isRequired,
-      slug: PropTypes.string,
-      id: PropTypes.number.isRequired,
-    }),
-  ),
-  updateSuggestions: PropTypes.func.isRequired,
+  getSuggestions: PropTypes.func.isRequired,
+  mode: PropTypes.string.isRequired,
   fullWidth: PropTypes.bool,
   disabled: PropTypes.bool,
   enableAdd: PropTypes.bool,
-  category: PropTypes.string.isRequired,
+  slugify(props, propName) {
```

Comment:
I love the custom more specific proptype check, awesome :D. For the function check you can use `_.isFunction` from lodash as well